iis-wf/iis-wf-ingest-pmc/src/test/resources/eu/dnetlib/iis/wf/ingest/pmc/metadata/sampledataproducer/data/fault.properties

```diff
@@ -0,0 +1,3 @@
+inputObjectId=50|od_______908::14ddacb589be0a68489f8981_invalid
+code=org.jdom.input.JDOMParseException
+message=Error on line 1: Content is not allowed in prolog.
```

Comment:
My intention was to describe expected file format, having `.properties` should indicate we are expecting `key=value` pairs. 

I could be more precise and go with e.g. `.expectations` which will be less informative in terms of file format but more precise in terms of the purpose. What do you think?